README.md

```diff
@@ -21,5 +21,114 @@ yarn run unit
+| project_samples  | age           | Age                                      | decimal     |                  | false       |             | false          | false         | true    | filters         |            | true     | 
+| project_samples  | sex           | Sex                                      | enum        |                  | false       | male,female | false          | true          | true    | filters         |            | false    | 
+| project_samples  | smoking       | Ever smoked                              | bool        |                  | false       |             | false          | true          | true    | filters         |            | false    | 
+| project_samples  | wbcc          | Whole bloodcell count                    | bool        |                  | false       |             | false          | true          | true    | data_type       |            | false    | 
+| project_samples  | wgs           | Whole Genome Sequencing (Illumina Hiseq) | bool        |                  | false       |             | false          | true          | true    | data_type       |            | false    | 
+| project_samples  | data_type     | Data type                                | compound    |                  | false       |             | false          | false         | true    | filters         |            | true     | 
+| project_samples  | methylome     | Methylome (Illumina 450K)                | bool        |                  | false       |             | false          | true          | true    | data_type       |            | false    | 
+| project_samples  | filters       | Filters                                  | compound    |                  | false       |             | false          | false         | true    |                 |            | true     | 
+| project_samples  | age_years     | Age in years                             | int         |                  | false       |             | false          | true          | true    | filters         | age        | false    | 
+| project_biobanks | id            | ID                                       | string      |                  | true        |             | false          | false         | true    |                 |            | true     | 
+| project_biobanks | abbr          | Abbreviation                             | string      |                  | false       |             | true           | false         | true    |                 |            | true     | 
+| project_biobanks | description   | Description                              | string      |                  | false       |             | false          | false         | true    |                 |            | true     | 
+
+Download the complete model [here](../../raw/master/example/example-model.xlsx) 
+   
+## Configuration
+Colors and attributes used to create the plots can be configured in the ```index.html```
+as follows:
+``` javascript
```

Comment:
Do not add a space between the backticks and the code style
And i think javascript is js